### PR TITLE
Feature/oas

### PR DESCRIPTION
**Pull request by edtu1**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/gp-connect-prescriptions-management-fhir.yaml [hunk 1]
```diff
@@ -549,73 +550,147 @@ paths:
...
-                          intent: plan
-                          medicationReference:
-                            reference: 8b67891-df9n-3nfu-jnvc8-18s08fvhs0
-                          subject: 
-                            reference:  Patient/9000000009
-                          authoredOn: 2018-01-06
-                          requester: 
-                            reference:  Patient/9000000009
-                          substitution: false
+                          id: b269d1d7-1acf-47bb-8b3c-e38b583d9a07
```
Comment (agreenhalgh) [inline specification/gp-connect-prescriptions-management-fhir.yaml:623]: the really important ID to include is the EPS prescription ID. In EPS this uses 
```
"identifier" : {
          "system" : "https://fhir.nhs.uk/Id/prescription-order-number",
          "value" : "CDF34E-A99968-4FF3BQ"
        }
```
Note the ID format which is UUID-like but not actualy a UUID.
Think we should say 'EPS R2 prescriptions will include EPS prescription ID which can be used to track the status of the prescription'.

**Review by agreenhalgh — CHANGES_REQUESTED** [review]
Two key issues:
- EPS prescription ID missing
- MedicationRequest retrieved by ID, not searched by patient

File: specification/gp-connect-prescriptions-management-fhir.yaml [hunk 2]
```diff
@@ -29,53 +29,55 @@ info:
...
 
-    Information governance (IG)
+    ####Information governance (IG)
     
     You must:
 
     * be compliant with the GP Connect Direct Care API Information Governance Model
 
-    * manage access to your system locally using role based access control. This does not need to be compliant with the national RBAC model and GP Connect products do not require smartcards to control access, though they can be used if already implemented be using the GP Connect APIs for direct care purposes for NHS patients in England
+    * manage access to your system locally using role based access control
```
[the remark this answers is not in the document]
Comment (edtu1) [reply specification/gp-connect-prescriptions-management-fhir.yaml:58]: Removed

File: specification/gp-connect-prescriptions-management-fhir.yaml [hunk 3]
```diff
@@ -394,16 +402,16 @@ paths:
...
-                          display: Continuous long term (repeat dispensing)
+                        - system: "https://fhir.hl7.org.uk/CodeSystem/UKCore-MedicationRequestCourseOfTherapy"
+                          code: continuous
+                          display: Continuous long term therapy
                     authoredOn: 2022-10-13T16:20:27+07:00
                     substitution:
                       allowedBoolean: false
                     dispenseRequest:
                       performer:
                         reference: Organization/Y12345
```
[the remark this answers is not in the document]
Comment (edtu1) [reply specification/gp-connect-prescriptions-management-fhir.yaml:411]: updated

File: specification/gp-connect-prescriptions-management-fhir.yaml [hunk 4]
```diff
@@ -549,73 +548,147 @@ paths:
...
+                            quantity: 2
+                            performer:
+                              reference: Organization/Y12345
+                              type: Organization
+                          dosageInstruction: 
+                          - text: One to two puffs to be inhaled as needed
+                            method:
+                               coding: 
+                                - system: "http://snomed.info/sct"
+                                  code: 420652005
```
[the remark this answers is not in the document]
Comment (edtu1) [reply specification/gp-connect-prescriptions-management-fhir.yaml:660]: Will need to check this.

**Review by agreenhalgh — APPROVED** [review]
OK, let's raise validation against structured dose syntax as a future task

**Pull request merged**